examples/get_iss_breast_data.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import io
+import os
+from typing import IO, Tuple
+
+from skimage.io import imread, imsave
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Indices
+from starfish.util.argparse import FsExistsType
+
+SHAPE = 1044, 1390
+
+
+class IssCroppedBreastTile(FetchedTile):
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return SHAPE
+
+    @property
+    def format(self) -> ImageFormat:
+        return ImageFormat.TIFF
+
+    @staticmethod
+    def crop(img):
+        crp = img[40:1084, 20:1410]
+        return crp
+
+    def tile_data_handle(self) -> IO:
+        im = self.crop(imread(self.file_path))
+        fh = io.BytesIO()
+        imsave(fh, im, plugin='tifffile')
+        fh.seek(0)
+        return fh
+
+
+class ISSCroppedBreastPrimaryTileFetcher(TileFetcher):
+    def __init__(self, input_dir):
+        self.input_dir = input_dir
+
+    @property
+    def ch_dict(self):
+        ch_str = ['Cy3 5', 'Cy3', 'Cy5', 'FITC']
+        ch = [2, 1, 3, 0]
+        ch_dict = dict(zip(ch, ch_str))
+        return ch_dict
+
+    @property
+    def hyb_dict(self):
+        hyb_str = ['1st', '2nd', '3rd', '4th']
+        return dict(zip(range(4), hyb_str))
+
+    def get_tile(self, fov: int, hyb: int, ch: int, z: int) -> FetchedTile:
+        filename = 'slideA_' + str(fov + 1) + '_' + \
+                   self.hyb_dict[hyb] + '_' + self.ch_dict[ch] + '.TIF'
+        file_path = os.path.join(self.input_dir, filename)
+        return IssCroppedBreastTile(file_path)
+
+
+class ISSCroppedBreastAuxTileFetcher(TileFetcher):
+    def __init__(self, input_dir, aux_type):
+        self.input_dir = input_dir
+        self.aux_type = aux_type
+
+    def get_tile(self, fov: int, hyb: int, ch: int, z: int) -> FetchedTile:
+        if self.aux_type == 'nuclei':
+            filename = 'slideA_' + str(fov + 1) + '_DO_' + 'DAPI.TIF'
+        elif self.aux_type == 'dots':
+            filename = 'slideA_' + str(fov + 1) + '_DO_' + 'Cy3.TIF'
+        else:
+            msg = 'invalid aux type: {}'.format(self.aux_type)
+            msg += ' expected either nuclei or dots'
+            raise ValueError(msg)
+
+        file_path = os.path.join(self.input_dir, filename)
+
+        return IssCroppedBreastTile(file_path)
+
+
+def format_data(input_dir, output_dir):
+    if not input_dir.endswith("/"):
```

Comment:
these should not be necessary.

Reply:
lol i copied this from your code in get_iss.py. i can nuke it.